src/components/credentials/credentialsTableCells.js

```diff
@@ -117,16 +120,16 @@ const sourcesContent = ({ [apiTypes.API_RESPONSE_CREDENTIAL_SOURCES]: sources }
 
   return (
     <List isPlain>
-      {updatedSources?.map(source => (
-        <ListItem
-          key={source[apiTypes.API_RESPONSE_CREDENTIAL_SOURCES_NAME]}
-          icon={
-            <ContextIcon symbol={ContextIconVariant[source[apiTypes.API_RESPONSE_CREDENTIAL_SOURCES_SOURCE_TYPE]]} />
-          }
-        >
-          {source[apiTypes.API_RESPONSE_CREDENTIAL_SOURCES_NAME]}
-        </ListItem>
-      ))}
```

Comment:
required:
confirm this syntax isn't shared with the sources and scans table cells

Reply:
confirmed